shadows/framework/src/main/java/org/robolectric/shadows/ShadowBitmap.java

```diff
@@ -899,4 +903,27 @@ void setBufferedImage(BufferedImage bufferedImage) {
   BufferedImage getBufferedImage() {
     return bufferedImage;
   }
+
+  void drawRect(RectF r, Paint paint) {
+    if (bufferedImage == null) {
+      return;
+    }
+    int[] pixels = ((DataBufferInt) bufferedImage.getRaster().getDataBuffer()).getData();
+
+    Graphics2D graphics2D = bufferedImage.createGraphics();
+    Rectangle2D r2d = new Rectangle2D.Float(r.left, r.top, r.right - r.left, r.bottom - r.top);
+    graphics2D.draw(r2d);
+    graphics2D.dispose();
+
+    double bottom=r2d.getY()+r2d.getHeight();
```

Comment:
Please follow https://github.com/robolectric/robolectric/wiki/Robolectric's-code-style to format code style locally.

Reply:
i had forgot to implement it..
done..